week03_model_free/README.md

```diff
@@ -33,12 +33,17 @@ Please pay attention for uncommenting first lines in code if you use Colab.
 
 ### ./seminar_py2
 _this assignment borrows code from awesome [cs188](http://ai.berkeley.edu/project_overview.html)_
-This homework assignment works on __python2 only__. If you stick to py3, consider seminar_alternative. Or just install it for this homework alone and remove afterwards.
+This homework assignment works on __python2 only__. If you stick to py3, just install it for this homework alone and remove afterwards.
 
-This homework also requires some physical display (e.g. laptop monitor). It won't work on binder VM / headless server. Please run it on laptop or consider ./seminar_alternative
+This homework also requires some display (e.g. laptop monitor). It won't work on binder VM / headless server.
 
+Solution ways:
```

Comment:
I do not pretend to be the best writer I just expressed my thoughts and try to do as less as possible edits. I suppose that it is work for the chiefs of the course to write the best final README. Or it can be done by a future contributor, if he feels itself as documentation specialists.
To sum it up, I will put a better verison if it is provided OR let it to be out of the PR